sky/engine/core/painting/PictureRecorder.cpp

```diff
@@ -34,21 +34,12 @@ PassRefPtr<Picture> PictureRecorder::endRecording()
     RefPtr<Picture> picture = Picture::create(
         adoptRef(m_pictureRecorder.endRecording()));
     m_canvas->clearSkCanvas();
+    m_canvas->ClearDartWrapper();
     m_canvas = nullptr;
+    ClearDartWrapper();
     return picture.release();
 }
 
-PassRefPtr<Drawable> PictureRecorder::endRecordingAsDrawable()
```

Comment:
I think we should leave it. I can see someone wanting to make a framework using Drawables rather than layers. We do need to test it, though. Can you file a bug for that?